### PR TITLE
MHR api transfer and location schema updates.

### PR DESCRIPTION
**Pull request by doug-lovett**
Signed-off-by: Doug Lovett <doug@diamante.ca>

*Issue #:* /bcgov/entity#13608

*Description of changes:*
- registration schema add documentDescription
- location schema add status, locationType
- add transfer schema and unit tests

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the registry-schemas license (Apache 2.0).

### CODE REVIEW

File: src/registry_schemas/version.py [hunk 1]
```diff
@@ -22,4 +22,4 @@
 Development release segment: .devN
 """
 
-__version__ = '1.2.0'  # pylint: disable=invalid-name
+__version__ = '1.6.2'  # pylint: disable=invalid-name
```
Comment (cameron-freshworks) [inline src/registry_schemas/version.py:25]: Big jump :)

**Review by cameron-freshworks — APPROVED** [review]
Looks good!

File: src/registry_schemas/version.py (hunk 1 above)
Comment (doug-lovett) [reply src/registry_schemas/version.py:25]: The schema has Git releases which this value was not reflecting. The next Git release is 1.6.2.

Comment (cameron-freshworks) [reply src/registry_schemas/version.py:25]: Nice! All caught up

**Pull request merged**